README.md

```diff
@@ -158,11 +158,14 @@ A simple TCP reverse proxy with SSL termination on port 993 and proxy protocol t
 					"routes": [
 						{
 							"handle": [
+								{
+									"handler": "proxy_protocol"
+								},
```

Comment:
With this change, the description of the example above should probably be adjusted to mention that `proxy_protocol` support for the incoming connection is included.